src/coffeescript.coffee

```diff
@@ -106,6 +100,7 @@ exports.compile = compile = withPrettyErrors (code, options = {}) ->
     ast.tokens = tokens
     return ast
 
+  nodes.findReferencedVars()
```

Comment:
What is this doing? Populating the `referencedVars` property of the root and all children? Can we add a comment explaining this?